<div>
    <button>up</button>
    <div>
        <button onclick="yawd()">left</button>
        <button onclick="reset1()">reset</button>
        <button onclick="yawi()">right</button>
    </div>
    <button>down</button>
</div>
<div>
    <button onclick="pitchi()">up</button>
    <div>
        <button onclick="rolld()">left</button>
        <button onclick="reset2()">reset</button>
        <button onclick="rolli()">right</button>
    </div>
    <button onclick="pitchd()">down</button>
</div>

<script>
    const url = window.location.origin
    const rolli = () => {
        fetch(`${url}/action?q=ri`)
            .then(response => {
                if (!response.ok) {
                    throw new Error('Network response was not ok');
                }
                return response.text();
            })
            .then(data => {
                console.log('Response:', data);
            })
            .catch(error => {
                console.error('Error:', error);
            });
    }
    const rolld = () => {
        fetch(`${url}/action?q=rd`)
            .then(response => {
                if (!response.ok) {
                    throw new Error('Network response was not ok');
                }
                return response.text();
            })
            .then(data => {
                console.log('Response:', data);
            })
            .catch(error => {
                console.error('Error:', error);
            });
    }
    const pitchi = () => {
        fetch(`${url}/action?q=pi`)
            .then(response => {
                if (!response.ok) {
                    throw new Error('Network response was not ok');
                }
                return response.text();
            })
            .then(data => {
                console.log('Response:', data);
            })
            .catch(error => {
                console.error('Error:', error);
            });
    }
    const pitchd = () => {
        fetch(`${url}/action?q=pd`)
            .then(response => {
                if (!response.ok) {
                    throw new Error('Network response was not ok');
                }
                return response.text();
            })
            .then(data => {
                console.log('Response:', data);
            })
            .catch(error => {
                console.error('Error:', error);
            });
    }
    const yawi = () => {
        fetch(`${url}/action?q=yi`)
            .then(response => {
                if (!response.ok) {
                    throw new Error('Network response was not ok');
                }
                return response.text();
            })
            .then(data => {
                console.log('Response:', data);
            })
            .catch(error => {
                console.error('Error:', error);
            });
    }
    const yawd = () => {
        fetch(`${url}/action?q=yd`)
            .then(response => {
                if (!response.ok) {
                    throw new Error('Network response was not ok');
                }
                return response.text();
            })
            .then(data => {
                console.log('Response:', data);
            })
            .catch(error => {
                console.error('Error:', error);
            });
    }

    const reset1 = () => {
        fetch(`${url}/action?q=r1`)
            .then(response => {
                if (!response.ok) {
                    throw new Error('Network response was not ok');
                }
                return response.text();
            })
            .then(data => {
                console.log('Response:', data);
            })
            .catch(error => {
                console.error('Error:', error);
            });
    }
    const reset2 = () => {
        fetch(`${url}/action?q=r2`)
            .then(response => {
                if (!response.ok) {
                    throw new Error('Network response was not ok');
                }
                return response.text();
            })
            .then(data => {
                console.log('Response:', data);
            })
            .catch(error => {
                console.error('Error:', error);
            });
    }
</script>